test/api/test_tools.py

```diff
@@ -465,10 +465,14 @@ def test_dynamic_list_output( self ):
         output_collection = get_collection_response.json()
         self._assert_has_keys( output_collection, "id", "name", "elements", "populated" )
         assert output_collection[ "populated" ]
-        assert len( output_collection[ "elements" ] ) == 2
         self.assertEquals( output_collection[ "name" ], "Table split on first column" )
 
-        # TODO: verify element identifiers
+        assert len( output_collection[ "elements" ] ) == 2
+        output_element_0 = output_collection["elements"][0]
+        assert output_element_0["element_index"] == 0
+        assert output_element_0["element_identifier"] == "samp1"
+        output_element_hda_0 = output_element_0["object"]
+        assert output_element_hda_0["metadata_column_types"] != None
```

Comment:
I think [flake8](https://travis-ci.org/galaxyproject/galaxy/jobs/189194240#L177) wants a `is not None` or `assert output_element_hda_0["metadata_column_types"]` here.